api/output_policy.go

```diff
@@ -44,8 +48,22 @@ func (d *OutputPolicyError) HCLString() (string, error) {
 
 // Builds a sample policy document from the request
 func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
```

Comment:
Do we have a test for this function? If not, I think it would be great to add one.

Reply:
There are not for this function but for the components that are within it. I can add some tests though.